migrations/3_register_vendors.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars */
+const SupplyChain = artifacts.require('SupplyChain');
+
+const vendors = [
+  '0xeceffab2caf1ec535d407d366d747b575018d65e',
+  '0xead172adb6cd5fe0b58150301ca25598c0dd1613',
+  '0x18b0de3577c6f97522b1335344ad3d05cf3a6e2c',
+  '0x75711d62a3db9f945ed901fea25f2ba6ae4cc5cc',
+];
+
+module.exports = async (deployer) => {
+  const supplyChain = await SupplyChain.deployed();
+  const vendorRegistrations = vendors.map(vendor => supplyChain.registerVendor(vendor));
```

Comment:
Are you sure that you can make these transactions concurrently? Wouldn't be any problem with nonce?

Reply:
TBH I haven't considered this when writing this code. To be absolutely sure I'd have to check the internal implementation of web3.js. But it did work on Ropsten: https://ropsten.etherscan.io/address/0xBAc15deF75fA2F4ec8B252faCF227Fb915470e8F. Can we agree to leave this as is? If it won't work during the workshops we'll be able to fix this very quickly.

Reply:
OK, we can do that.